plugins/reach/lib/api/data/vendorCatalogItem/KalturaVendorDocumentEnrichmentCatalogItem.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ */
+class KalturaVendorDocumentEnrichmentCatalogItem extends KalturaVendorCatalogItem
+{
+	/**
+	 * @var KalturaVendorDocumentEnrichmentType
+	 */
+	public $documentEnrichmentType;
+
```

Comment:
Need to add $map_between_objects

and function getMapBetweenObjects

(Look at KalturaVendorCaptionsCatalogItem.php)